skill-tree/src/main/java/com/RDS/skilltree/Endorsement/EndorsementController.java

```diff
@@ -79,4 +79,10 @@ public ResponseEntity<GenericResponse<EndorsementDTO>> postEndorsement(
                 new GenericResponse<EndorsementDTO>(null, "Failed to create endorsement"),
                 HttpStatus.BAD_REQUEST);
     }
+
+    @PatchMapping(value = "/{id}")
+    public ResponseEntity<GenericResponse<Void>> updateEndorsementStatus(
+            @PathVariable(value = "id") String id, @RequestParam String status) {
```

Comment:
why Id is string here

Reply:
Updated to UUID [here](https://github.com/Real-Dev-Squad/skill-tree-backend/pull/116/commits/99349dce4a1b5c1cbe38d13899a3941e1357ffd1)
After some research, I found out about [StringToUUIDConverter](https://github.com/spring-projects/spring-framework/blob/master/spring-core/src/main/java/org/springframework/core/convert/support/StringToUUIDConverter.java)